### DISCUSSION TITLE
关于对软件的若干建议

### DISCUSSION THREAD

**Discussion by Huaweidev** [lt94/MovieHeavens#31] [Ideas]
#### 关于对软件的若干建议
1. 主界面上的“查询内容：查询未开始”字样建议放在状态栏，不然拉大界面后，比例就不对了，见下图。
2. 建议开放这个repo的wiki页面，以便于大家对产品有心得可以写在里面，不用开issue。
    * 例如我想分享给大家，[“https://ygdy8.com”阳光电影网](https://ygdy8.com)的下载连接```直链```和```磁力链接```用IDM或者Aria下载根本没速度。
![pic_0129201736](https://user-images.githubusercontent.com/10258421/51907857-0e019f80-2403-11e9-8c69-68f8b4f7d6f3.jpg)

**Comment by LeetaoGoooo** [comment]
感谢建议，界面这一块，我对qt不是很熟，当时也是练手，你有兴趣的话可以提交 PR，wiki 我晚点开放，最后电影天堂的链接感觉多数都是迅雷的链接

发件人： Jeff Li
发送时间： 2019-01-29 20:24:57
收件人：  lt94/MovieHeavens
抄送：  Subscribed
主题： [lt94/MovieHeavens] 关于对软件的若干建议 (#9)
关于对软件的若干建议

主界面上的“查询内容：查询未开始”字样建议放在状态栏，不然拉大界面后，比例就不对了，见下图。
建议开放这个repo的wiki页面，以便于大家对产品有心得可以卸载里面，不用开issue。
例如我想分享给大家，“https://ygdy8.com”阳光电影网的下载连接直链和磁力链接用IDM或者Aria下载根本没速度。
